support-docs/README.md

```diff
@@ -244,10 +244,342 @@ The table below gives a description of the properties used to specify a switchin
 |package|Package definition of the switching regulator|Object| |
 |complianceList|List of compliances met by the switching regulator component|Array| |
 
+### 3.10	 Hardware
 
+TODO: not sure what separates this category from others? "hardware" is a bit generic sounding.
 
+####  3.10.1	 Switch
 
+The table below gives a description of the properties used to specify a switch in a digital datasheet.
 
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|definitions.json#/componentID|Yes|

```

Comment:
JSON type is Object. In general, I've noticed that all the conversion tool is adding the reference file reference as the JSON type entry for all the properties using $ref.

Reply:
Sorry, I meant to fix this before sending out because I had noticed it! I changed them all to Object for this CL.

In a future version, I think we should consider having something than Object since the type a part of the specification and might as well be documented. And with the automated script to generate this README, it is easy to change everything all at once.